src/Core/Command/DeadLetterRetryCommand.php

```diff
@@ -0,0 +1,243 @@
+<?php declare(strict_types=1);
+/**
+ * This File is part of JTL-Software
+ *
+ * User: marius
+ * Date: 7/23/21
+ */
+
+namespace JTL\SCX\Lib\Channel\Core\Command;
+
+use DateTimeImmutable;
+use JTL\Nachricht\Contract\Serializer\MessageSerializer;
+use JTL\Nachricht\Contract\Transport\Amqp\AmqpQueueLister;
+use JTL\Nachricht\Serializer\Exception\DeserializationFailedException;
+use JTL\Nachricht\Transport\Amqp\AmqpHttpConnectionFailedException;
+use JTL\Nachricht\Transport\Amqp\AmqpTransport;
+use JTL\SCX\Lib\Channel\Contract\Core\Log\ScxLogger;
+use JTL\SCX\Lib\Channel\Event\AbstractEvent;
+use PhpAmqpLib\Message\AMQPMessage;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DeadLetterRetryCommand extends AbstractCommand
+{
+    protected static $defaultName = 'chn:dead-letter:retry';
```

Comment:
Naming würde ich hier noch verbessern

Vorschlag 1
chn:message.retry

````
Available commands:
  help                                      Display help for a command
  list                                      List commands
 chn
  chn:message.consume                       Subscribe to all existing Event Queues and consume Messages from RabbitMQ
  chn:message.retry                         Moves messages from the Dead Letter queue back to their original queues
```` 

Vorschlag 2:
queue:dead-letter.retry

````
Available commands:
  help                                      Display help for a command
  list                                      List commands
 chn
  chn:message.consume                       Subscribe to all existing Event Queues and consume Messages from RabbitMQ
 db
  db:schema-up                              Migrate MongoDB Schema (run verbose to see index results)
 helper
  helper:emit.ChannelNotification           Helper command to emit Channel Notification with severity INFO
  // ...
 queue
  queue:dead-letter.retry                   Moves messages from the Dead Letter queue back to their original queues
 scx-api
  scx-api:get.events                        Consume seller events created from polling the SCX Channel API
````

In beiden Fällen würde es sich besser eingliedern